config/config.go

```diff
@@ -143,14 +143,14 @@ func (c *Config) IsAzure() bool {
 	return isAzureHost || c.AzureResourceID != ""
 }
 
-// IsGcp returns true if client is configured for GCP
+// IsGcp returns if the client is configured for Databricks on Google Cloud.
 func (c *Config) IsGcp() bool {
 	return strings.Contains(c.Host, ".gcp.databricks.com")
 }
 
-// IsAws returns true if client is configured for AWS
+// IsAws returns if the client is configured for Databricks on AWS.
 func (c *Config) IsAws() bool {
-	return !c.IsAzure() && !c.IsGcp()
+	return strings.Contains(c.Host, ".databricks.com") && !c.IsAzure() && !c.IsGcp()
```

Comment:
Seems fine, one other way which is more of a whitelist approach is
```go
strings.Contains(c.Host, ".dev.databricks.com") || strings.Contains(c.Host, ".cloud.databricks.com")
```
staging is a subzone of prod.